packages/element/src/create-interpolate-element.js

```diff
@@ -0,0 +1,326 @@
+/**
+ * External dependencies
+ */
+import { createElement, Fragment } from 'react';
+
+let indoc,
+	offset,
+	output,
+	stack,
+	keyIndex;
+
+/**
+ * Matches tags in the localized string
+ *
+ * This is used for extracting the tag pattern groups for parsing the localized
+ * string and along with the map converting it to a react element.
+ *
+ * There are four references extracted using this tokenizer:
+ *
+ * match: Full match of the tag (i.e. <strong>, </strong>, <br/>)
+ * isClosing: If the match has a closing tag (i.e. </)
+ * name: The name portion of the tag (strong, br) (if )
+ * isSelfClosed: Whether the match is for a self-closing tag (i.e. />)
```

Comment:
it may be more accurate to describe these as something like "the closing slash, if it exists" because it either returns a string containing `/` or it doesn't return a value for it

Reply:
addressed in d59f4b6